src/IW/Server/Issue.hs

```diff
@@ -7,16 +7,16 @@ module IW.Server.Issue
        , issuesHandler
        ) where
 
-import IW.Core.Issue (Issue (..))
+import IW.Core.Issue (Issue (..), Label (..))
 import IW.Core.WithId (WithId (..))
-import IW.Db (WithDb, getIssues, getIssuesByLabel)
+import IW.Db (WithDb, getIssues, getIssuesByLabels)
 import IW.Server.Types (AppServer, ToApi)
 
 
 newtype IssueSite route = IssueSite
     { issuesRoute :: route
         :- "issues"
-        :> QueryParam "label" Text
+        :> QueryParam "labels" Text
```

Comment:
If you want to pass the list of labels, I propose to replace `QueryParam` with `ReqBody` for simplicity. So the list of labels will be given as a JSON request body. For now, we have the only list of labels, but down the road, we might have a complicated filtering structure. So it's easier just to pass it a body.

```suggestion
        :> ReqBody '[JSON] [Label]
```

Reply:
@chshersh I see :bulb: